src/book.js

```diff
@@ -841,8 +810,17 @@ export class Book {
         console.log("After creating signatures, our filelist looks like: ",this.filelist)
     }
 
+    bundleSettings() {
+        const currentConfig = loadConfiguration();
+        const settings = `Imposer settings: ${JSON.stringify(currentConfig, null, 2)}`
+            + '\n\n'
+            + `Link to the imposer with these settings: ${window.location.href}`
+        this.zip?.file("settings.txt", settings);
```

Comment:
.... I don't get this file? was about to leave a comment that the `.url` file is nice enough, but was hoping for a `.txt` -- but now I see this snippet? Am not seeing it in my downloaded zip...?

Reply:
my bad-- totes works on the correct branch!